docs/getting-started/sync-with-github.md

```diff
@@ -1,61 +1,181 @@
 
-If you are hosting a local Amplication server, you should first read [Connect Amplication server to GitHub](https://docs.amplication.com/running-amplication-platform/connect-server-to-github/) to see how to configure the Server to work with a new GitHub application.
+Click on the _Connect_ button for GitHub.
 
-## Using the Amplication hosted service
+:::important
+If you're already connected to GitHub, you can either add another GitHub organization or select an account you connected previously.
 
-If you are using the hosted service on [https://app.amplication.com](https://app.amplication.com/), you can continue with this article.
+You can then go to [selecting your repository](#select-your-repository).
+:::
+
+## Install Amplication's GitHub App
+
+Now the _Install Amplication GitHub App_ page opens.
+Select your preferred account or organization on GitHub.
+
+![](./assets/sync-with-git-provider/dac2b208-365c-4bf4-9a49-f0914648cf8f.png)
 
-## Authorize Amplication to access your GitHub account[](https://docs.amplication.com/sync-with-git-provider/#authorize-amplication-to-access-your-github-account)
+### Set Amplication's Configuration Settings
 
-Before integrating your application with GitHub, you need to authorize Amplication to get access to your GitHub account.
+After selecting your preferred account or organization, you'll be led to the configuration page.
+You can configure the GitHub App to only allow access to specific repositories in your account, or allow Amplication to access all of your repositories.
 
-1. Open up an Amplication project.
-2. Click **Connect to GitHub** at the bottom left of your project. The _Sync with GitHub_ screen opens.
+![](./assets/sync-with-git-provider/32086706-520a-4996-8647-ecd81546f4f5.png)
 
-![](./assets/sync-with-git-provider/sync-with-github.png)
+After selecting your repositories, click on the `Next` button.
+You'll be brought back to Amplication.
 
-If this is the first time that you have connected to GitHub, the **Connect to GitHub** button is displayed.
-If you have already connected to GitHub, you will be prompted to choose the installed organization. If several organizations are configured, you will be prompted to select the required organization.
+## Select Your Repository
 
-3. Click **Connect to GitHub**
-   The _Install Amplication Github App_ opens.
+You'll see a modal appear with a populated list of repositories based on your configuration settings from above.
 
-4. Select the GitHub account you want to use.
-5. In the new window, select on which repositories you want to install the app.
+![](./assets/first-service/select-repository.png)
+
+:::important
+When you select a repository, make sure that it's not an empty repository. It should have at least one file in it.
+:::
 
 :::note
 Currently, only by connecting to an organization can you create a new GitHub Repository. A User can sync only with an existing repository.
```

Comment:
"A User can sync only with an existing repository." I didn't understand this sentence
I think maybe you meant that a personal git account can only sync to existing repositories and cannot create a new one via Amplication. FYI - We are adding a button for Personal Git accounts so that GitHub will open up where they can create a new repo.